bin/sozo/src/commands/migrate.rs

```diff
@@ -75,15 +81,50 @@ impl MigrateArgs {
             let MigrationResult { manifest, has_changes } =
                 migration.migrate(&mut spinner).await.context("Migration failed.")?;
 
+            let ipfs_url = ipfs.url(profile_config.env.as_ref());
+            let ipfs_username = ipfs.username(profile_config.env.as_ref());
+            let ipfs_password = ipfs.password(profile_config.env.as_ref());
+
+            let mut metadata_upload_text = String::new();
+
+            if ipfs_url.is_some() && ipfs_username.is_some() && ipfs_password.is_some() {
+                let mut metadata_service = IpfsMetadataService::new(
+                    &ipfs_url.unwrap(),
+                    &ipfs_username.unwrap(),
+                    &ipfs_password.unwrap(),
+                )?;
```

Comment:
To simplify this part, we may have the `IpfsMetadataService` that can be initialized from the `profile_config`?
This makes the service a little more tight to the dojo configuration, and if later we want to isolate the IPFS in an other crate, the extended implementation can remain into `dojo-world`, wdyt?

```rust
let metadata_service: IpfsMetadataService = profile_config.env.as_ref().try_into()?;
```